doc/source/whatsnew/v1.0.0.rst

```diff
@@ -110,6 +110,7 @@ Other enhancements
 - :meth:`DataFrame.to_json` now accepts an ``indent`` integer argument to enable pretty printing of JSON output (:issue:`12004`)
 - :meth:`read_stata` can read Stata 119 dta files. (:issue:`28250`)
 - Added ``encoding`` argument to :func:`DataFrame.to_html` for non-ascii text (:issue:`28663`)
+- :meth:`Styler.format_null` is now added into the built-in functions to help formatting missing values (:issue:`28358`)
```

Comment:
can u add this into the user guide as well

Reply:
I'd like the name ``format_nans`` better, to be similar to ``fillna``, ``hasnans`` etc.

@jreback?

Reply:
Sure, I'll be glad to!